addons/spreadsheet/static/src/helpers/model.js

```diff
@@ -163,7 +164,9 @@ function containsOdooFunction(content) {
     if (
         !content ||
         !content.startsWith("=") ||
-        (!content.toUpperCase().includes("ODOO.") && !content.toUpperCase().includes("_T"))
+        (!content.toUpperCase().includes("ODOO.") &&
+            !content.toUpperCase().includes("_T") &&
+            !content.toUpperCase().includes("PIVOT"))
```

Comment:
Function name is confusing now, `containsOdooFunction` returns true if there is a non-Odoo pivot function in the content. Maybe the check thatthe pivot type is Odoo should be done here ?

Reply:
If I do the check here, I have to give this function the model, the sheet and the xc and change the code to make something way longer:
```js
return iterateAstNodes(ast).some((ast) => {
    if (ast.type !== "FUNCALL") {
        return false;
    }
    if (
        ast.value.toUpperCase().startsWith("ODOO.") ||
        ast.value.toUpperCase().startsWith("_T")
    ) {
        return true;
    }
    if (ast.value.toUpperCase().startsWith("PIVOT")) {
        // check if the pivot is an odoo pivot
        const { col, row } = toCartesian(xc);
        const position = { sheetId, col, row };
        const pivotId = model.getters.getPivotIdFromPosition(position);
        return pivotId && model.getters.getPivotDefinition(pivotId).type == "ODOO";
    }
});
```

Is it ok for you to rename this function (only used in this file) to `containsOdooCandidateFunctions` or `containsPossibleLoadedFunctions`, or something like that ?